File: pset6/mario/mario.py
from cs50 import get_int

# Get integer between 1-8 from user
while True:
    height = get_int("Height: ")
    if (height > 0 and height < 9):
        break

for i in range(height):
    # Print empty spaces (w/out newline)
    print(" " * ((height - i) - 1), end="")
    # Print block(s) (w/out newline)
    print("#" * (i + 1), end="")
    # Print new line
    print()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // initialise height 
    int height;
    
    // prompt user for input 
    do 
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    
    for (int i = 0; i < height; i++)
    {
        // print co
        for (int j = 0; j < (height - i) - 1; j++) 
        {
            printf(" ");
        }
        for (int k = 0; k < (i + 1); k++) 
        {
            printf("#");
        }
        // print new line after both inner loops finish
        printf("\n");
    }
    
}
